util/topsql/tracecpu/profile.go

```diff
@@ -43,15 +43,19 @@ const (
 // GlobalSQLCPUProfiler is the global SQL stats profiler.
 var GlobalSQLCPUProfiler = newSQLCPUProfiler()
 
-// Collector uses to collect SQL execution cpu time.
-type Collector interface {
+// Reporter uses to collect SQL execution cpu time.
+type Reporter interface {
 	// Collect uses to collect the SQL execution cpu time.
 	// ts is a Unix time, unit is second.
-	Collect(ts int64, stats []SQLCPUResult)
+	Collect(ts uint64, stats []TopSQLCPUTimeRecord)
 }
 
-// SQLCPUResult contains the SQL meta and cpu time.
-type SQLCPUResult struct {
+// TopSQLCPUTimeRecord represents a single record of how much cpu time a sql plan consumes in one second.
+//
+// PlanDigest can be empty, because:
+// 1. some sql statements has no plan, like `COMMIT`
+// 2. when a sql statement is being compiled, there's no plan yet
+type TopSQLCPUTimeRecord struct {
```

Comment:
How about rename `TopSQLCPUTimeRecord` to `SQLCPUTimeRecord`? 
Since `tracecpu` pkg shouldn't know `TopSQL`.

Reply:
LGTM, fixed